nymms/schemas/__init__.py

```diff
@@ -105,12 +117,43 @@ def increment_attempt(self):
 
 class Result(StateModel, OriginModel):
     id = StringType(required=True)
-    timestamp = TimestampType(default=time.time)
+    timestamp = TimestampType(default=arrow.get)
     output = StringType()
     task_context = JSONType()
 
```

Comment:
BTW, was that in your original pull request, or is that something you've found since then?